docs/guide/custom_policy.rst

```diff
@@ -85,6 +96,22 @@ using ``policy_kwargs`` parameter:
   model = PPO.load("ppo_cartpole", env=env)
 
 
+In the previous example, as CartPole's action space has a dimension of 2, the final dimensions of the ``net_arch``'s layers will be:
+
+
+.. code-block:: none
+
+            obs
+            <4>
+     /                \
+    <32>             <32>
+     |                 |
+    <32>             <32>
+     |                 |
+    <2>               <1>
+   action            value
+
+
```

Comment:
I think merging the comment above the code block with this one below would be clearer. :

```
.. note::
    An extra linear layer will be added on top of the layers specified in ``net_arch``, in order to have the right output dimensions and activation functions (e.g. Softmax for discrete actions).

    In the previous example, as CartPole's action space has a dimension of 2, the final dimensions of the ``net_arch``'s layers will be:


    .. code-block:: none
                obs
                <4>
        /                \
        <32>             <32>
        |                 |
        <32>             <32>
        |                 |
        <2>               <1>
      action            value

```

Reply:
Ok, did it.
I put the note on top of the example, because I thought it was more clear, still I merged the comment and the note